src/Symfony/Component/Ldap/Exception/DriverException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Ldap\Exception;
+
+/**
+ * LdapException is throw if php ldap module is not loaded.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class LdapException extends \RuntimeException
```

Comment:
You should create an empty `ExceptionInterface` marker like we do for almost all other components so that all exceptions in the component can implement it.